src/main/java/org/folio/circulation/domain/validation/ServicePointAccessValidator.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.HttpResult.succeeded;
+
+import java.util.function.Function;
+
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.support.HttpResult;
+import org.folio.circulation.support.ValidationErrorFailure;
+
+public class ServicePointAccessValidator {
```

Comment:
This class appears to be unused. Is that because the scope has been moved to https://issues.folio.org/browse/CIRC-150?

If so, was it decided to include these as placeholders for later development?

Reply:
Yes this is in place for https://issues.folio.org/browse/CIRC-150, and there are comments in the code to that effect.